src/components/User/CollabBadge.tsx

```diff
@@ -11,12 +11,14 @@ import { getUserName, isUserVerified } from "../../utils/user"
 
 interface Props extends IEntityBadgeProps {
   user: Collaboration
+  centered: boolean 
```

Comment:
I believe the centered prop should be optional, otherwise there will be compilation errors

Reply:
dang pushed the fix